rasa/utils/tensorflow/rasa_layers.py

```diff
@@ -0,0 +1,865 @@
+import tensorflow as tf
```

Comment:
What's the conceptual difference between `rasa_layers.py` and `layers.py`? Can we merge them? If not, can we rename `layers.py` to be more descriptive?

Reply:
`layers.py` are meant to be general keras layers that should be possible to use with any keras model (actually not sure that it's the case due to `inputs`, but that's the intention)
`rasa_layers.py` are specific and can only be used inside rasa model

Reply:
@dakshvar22 the model config is perhaps the most important thing that ties `rasa_layers` very strongly to Rasa models.

Reply:
It follows from the fact that these layers are really just pieces of TED/DIET factored out into separate components.

Reply:
Hmm, if I understand correctly most of the layers inside `rasa_layers` are composed of smaller layers coming from either `rasa_layer.py` or `layer.py`. They are not really "atomic" layers?

Reply:
@dakshvar22 exactly. I said "pieces of TED/DIET" but maybe I should've said "big chunks". They really aren't atomic. They sit somewhere between the basic components from `layers` and the overal architecture of DIET/TED.